### DISCUSSION TITLE
Align custom renovate image tags with renovate releases

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#22637] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I am building a custom renovate image via gitlab ci job. I'd like to align the image tag with the renovate release. 
I probably could use the gitlabci manager somehow or a regex manager. 

### What you tried so far.

Googled for examples to get inspired

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Is the image tag stored in the repo?

If instead it's generated by build scripts then the script could possibly read a `package.json` or `Containerfile` containing the Renovate version and then use that as the tag

**Reply by lmilbaum** [reply]
It is tracked in the Dockerfile as an ENV variable. I can pull it from there or just pull it from the same place I update the Dockerfile.

**Reply by rarkins** [reply]
Sounds like a regex manager could work - point it to the renovate npm or Docker published releases

**Reply by lmilbaum** [reply]
Yup. Good idea. Thanks.
